README.md

```diff
@@ -98,6 +98,40 @@ Your program should say "This could add vlan to eth0" or do anything you would
 want it to do :)
 
 
+Starting a switch from the command line
+=======================================
+
+```shell
+    pip install fake-switches
+    
+    # fake-switches <model> <listen_host> <listen_port>
+    fake-switches cisco_generic 0.0.0.0 22222
```

Comment:
fixed by telling people to open up a different shell (this is always easier than telling them how to get the process back with fg and dealing with port already in use errors)